OpenCore-BS.sh

```diff
@@ -29,7 +29,7 @@ OVMF_DIR="."
 args=(
   # This causes high cpu usage on the *host* side
   # -enable-kvm -m 3072 -cpu # Penryn,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,hypervisor=off,vmx=on,kvm=off,"$MY_OPTIONS"
-  -enable-kvm -m "$ALLOCATED_RAM" -cpu Penryn,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,"$MY_OPTIONS"
+  -enable-kvm -m "$ALLOCATED_RAM" -cpu host,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,"$MY_OPTIONS"
```

Comment:
Was this change tested end-to-end? Did Big Sur Beta 4 boot using this changed script and the changed config.plist file?

Reply:
I cannot test Big Sur on my system due to local constraints (I've switched back to bare metal, I tested it on Catalina and Mojave but that was a while ago)

Reply:
I recommend adding these CPU + associated config.plist changes to `boot-passthrough.sh` or a new script. Create a new folder for the new config.plist. Let's leave the rest of the working scripts as they are.

Reply:
The default scripts cannot fail. They have to work 100%.